pkg/apiserver/api_installer.go

```diff
@@ -336,9 +337,10 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 			}
 			route := ws.GET(action.Path).To(handler).
 				Filter(m).
-				Doc("read the specified " + kind).
-				Operation("read" + kind).
+				Doc("read the specified "+kind).
+				Operation("read"+kind).
```

Comment:
I think this file got checked in non-gofmt'd somehow, and since I changed it, it now forced me to gofmt it.

We only run gofmt on changes afaik.

(anyway, the change itself was due to gofmt, I didn't make the changes myself)